Client/mods/deathmatch/logic/CResourceModelStreamer.cpp

```diff
@@ -106,11 +106,15 @@ void CResourceModelStreamer::ReleaseAll()
 
 void CResourceModelStreamer::FullyReleaseModel(std::uint16_t modelId)
 {
-    std::uint16_t &refsCount = m_requestedModels[modelId];
+    auto refs = m_requestedModels.find(modelId);
+    if (refs == m_requestedModels.end())
+        return;
+
+    std::uint16_t refsCount = refs->second;
 
     if (refsCount > 0)
     {
-        refsCount = 0;
+        m_requestedModels[modelId] = 0;
```

Comment:
Please use the iterator or change `refsCount` to be a reference, to change the value here. The compiler will generate more code than necessary otherwise: https://godbolt.org/z/36f4a8Koe